Harvest.Web/ClientApp/src/Invoices/InvoiceTable.tsx

```diff
@@ -16,9 +17,9 @@ export const InvoiceTable = (props: Props) => {
       {
         Cell: (data: Cell<Invoice>) => (
           <div>
-            <a href={`/invoice/details/${data.row.original.projectId}/${data.row.original.id}`}>
+            <Link to={`/invoice/details/${data.row.original.id}`}>
```

Comment:
@mxthu313 You need the projectId in here too

Reply:
thanks for the catch.  sorry about that, fingers must have slipped.